bundle/DependencyInjection/Configuration.php

```diff
@@ -29,6 +29,15 @@ public function getConfigTreeBuilder()
                         ->prototype('scalar')->end()
                     ->end()
                 ->end()
+                ->arrayNode('template_theme_paths')
+                    ->useAttributeAsKey('theme')
+                    ->example(['some_theme' => ['/var/foo/some_theme_dir', '/another/theme/dir']])
+                    ->info('Collection of template paths by theme.')
```

Comment:
Actually, I'd make templates plural, and update the setting name. I know we've spent time on it, but it sounds wrong now :)

Funnily enough, it is reversed in the container parameter's name: `ezdesign.templates_path_map` 😉